ImportXml.py

```diff
@@ -0,0 +1,182 @@
+# flake8: noqa
```

Comment:
haha, sorry, little embarrasing, you can see in the code, because there are different type of questions, categories, and also have different tags, corresponding to various attributs, and also subquetions, if I want to parse all of these out, I really need loops with many if conditions. I try to make simplificaitons by using the dict comprehension, but because colon problelm in dictionary comprehension, I mean if I want to use dict comprehension, it cannot write like
```python
{a:b if condition else c:d for...}
```
it is incorrect, so I realy don't know how to simplify it. So I bypass the flake8 check.

Reply:
Don't worry we will simplify it together, so you can turn `flake8` on :)

Reply:
thank you.